packages/in_app_purchase/in_app_purchase_ios/example/pubspec.yaml

```diff
@@ -0,0 +1,33 @@
+name: in_app_purchase_ios_example
+description: Demonstrates how to use the in_app_purchase_ios plugin.
+publish_to: none
+
+dependencies:
+  flutter:
+    sdk: flutter
+  shared_preferences: ^2.0.0-nullsafety.1
+  in_app_purchase_ios:
+    # When depending on this package from a real application you should use:
+    #   in_app_purchase: ^x.y.z
+    # See https://dart.dev/tools/pub/dependencies#version-constraints
+    # The example app is bundled with the plugin so we use a path dependency on
+    # the parent directory to use the current plugin's version.
+    path: ../
+
+  # TODO(mvanbeusekom): Replace with pub.dev version when in_app_purchase_platform_interface is published
+  in_app_purchase_platform_interface:
+    path: ../../in_app_purchase_platform_interface
```

Comment:
replace with 1.0.0 as the platform interface has published

Reply:
Done